webpack.dev.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
   context: path.join(__dirname, "assets"),
   entry: {
     app: ["./js/app"],
-    join: ["./js/join"],
+    join: ["./js/join", "./js/location"],
```

Comment:
we can create a separate location JS file since the join JS file already has a lot of code

Reply:
Since I already have a separate js file called location.js, you're saying inside the webpack file have it as location: ["./js/location"]?

Reply:
thats correct!